components/catalogs/back/src/Infrastructure/Persistence/Family/GetFamilyLabelByCodeAndLocaleQuery.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Catalogs\Infrastructure\Persistence\Family;
+
+use Akeneo\Catalogs\Application\Persistence\Family\GetFamilyLabelByCodeAndLocaleQueryInterface;
+use Akeneo\Pim\Structure\Component\Model\FamilyInterface;
+use Akeneo\Tool\Component\StorageUtils\Repository\SearchableRepositoryInterface;
+
+/**
+ * @copyright 2023 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+final class GetFamilyLabelByCodeAndLocaleQuery implements GetFamilyLabelByCodeAndLocaleQueryInterface
+{
+    /** @var array<string,array<string,string>> $familyLabelByCode */
+    private array $familyLabelByCode = [];
```

Comment:
Normally, it's the same instance of the service that is injected in all places but to be completely sure your cache is efficient, make it static.

```suggestion
    private static array $familyLabelByCode = [];
```

Reply:
Talked via slack, we will keep the non static array